apps/google-analytics-4/lambda/src/lib/verify-signed-request-middleware.spec.ts

```diff
@@ -1,26 +1,59 @@
 import chai, { expect } from 'chai';
 import chaiHttp from 'chai-http';
-import sinon from 'sinon';
-import * as NodeAppsToolkit from '@contentful/node-apps-toolkit';
 import app from '../app';
+import sinon from 'sinon';
 
 chai.use(chaiHttp);
 
 describe('verifySignedRequestMiddleware', () => {
-  const verifyRequestStub = sinon.stub();
+  // to make this integration request, these header values were captured
+  const signature = '23b4145cf37d48030cb2818d3edbd2086f2566dcbf22c79bfbead3301fd4562a';
+  const timestamp = '1675203823305';
+  const spaceId = '30x8uoqewkkz';
+  const environmentId = 'master';
+  const userId = '6iUaAfJg2ZoKe4oEYF7kxe';
 
   beforeEach(() => {
-    // TODO: set headers and fully test signature verification later
-    sinon.stub(NodeAppsToolkit, 'verifyRequest').get(() => verifyRequestStub);
+    process.env.STAGE = 'dev'; // don't change this without recomputing the signed request signature
+    process.env.SIGNING_SECRET = 'mFjgMwlcE5iHKl855rGk9adWTP9ZagSamQctAElpptMtpGQmNIuGoRWOZZ_lhU_4';
```

Comment:
Wouldn't the clean solution be to pass in the env variables via a function parameter. Why not do that instead?